src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/Timestamp.cs

```diff
@@ -133,7 +133,7 @@ public Timestamp(SignedCms timestampCms)
             flags |= VerificationUtility.ValidateTimestamp(this, signature, treatIssueAsError, issues, SigningSpecifications.V1);
             if (flags == SignatureVerificationStatusFlags.NoErrors)
             {
-                issues.Add(SignatureLog.InformationLog(string.Format(CultureInfo.CurrentCulture, Strings.TimestampValue, GeneralizedTime.LocalDateTime.ToString()) + Environment.NewLine));
+                issues.Add(SignatureLog.InformationLog(string.Format(CultureInfo.CurrentCulture, Strings.TimestampValue, GeneralizedTime.LocalDateTime.ToString(CultureInfo.CurrentCulture)) + Environment.NewLine));
```

Comment:
There seems no behavior change. I think this is expected?
`string.Format(CultureInfo.CurrentCulture, s1, dateTime.ToString())` is equal to 
`string.Format(CultureInfo.CurrentCulture, s1, dateTime.ToString(CultureInfo.CurrentCulture))`, right?